pages/docs/connect/api-reference/api-fundamentals/index.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: "Overview"
+description: "Programmatically send bank transfers with our developer API."
+group:
+  title: API Fundamentals
+  weight: 0
+guide:
+  step: 0
+meta:
+  title: "API Reference | Dwolla Connect API Documentation"
+  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+---
+
+import Card from "../../../../../app/components/base/Card";
+import CardGrid from "../../../../../app/components/base/CardGrid";
+
+# Introduction
+
+Welcome to the Dwolla API documentation, your gateway to seamlessly integrating your software with robust banking infrastructure. Our API empowers developers with the essential tools to facilitate account-to-account payments, digital wallet functionality, customer identity verification, and bank account verification in a secure and efficient manner.
```

Comment:
Suggested edit:
Welcome to the Dwolla Connect API documentation! Dwolla Connect is a modern RESTful API solution that allows businesses to integrate account-to-account payment processing into their applications using their existing commercial bank accounts. Implementing and maintaining Dwolla’s unified API across financial institutions and payment modalities can save businesses a significant amount of development time and resources, unlock the ability to automate manual processes and increase visibility into payment activity. This API provides a comprehensive set of resources and endpoints for managing users, adding payment accounts, initiating transfers, and querying transaction history.